src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -44,30 +45,70 @@ public EasyPostClient(String apiKey) throws MissingParameterError {
         this(apiKey, Constant.DEFAULT_CONNECT_TIMEOUT_MILLISECONDS);
     }
 
+    /**
+     * EasyPostClient constructor.
+     *
+     * @param apiKey  API key for API calls.
+     * @param apiBase API base for API calls.
+     * @throws MissingParameterError
+     */
+    public EasyPostClient(String apiKey, String apiBase) throws MissingParameterError {
+        this(apiKey, Constant.DEFAULT_CONNECT_TIMEOUT_MILLISECONDS, Constant.DEFAULT_READ_TIMEOUT_MILLISECONDS,
+                apiBase);
+    }
+
     /**
      * EasyPostClient constructor.
      *
      * @param apiKey                     API key for API calls.
-     * @param connectTimeoutMilliseconds timeout for connection.
+     * @param connectTimeoutMilliseconds Timeout for connection.
      * @throws MissingParameterError
      */
     public EasyPostClient(String apiKey, int connectTimeoutMilliseconds) throws MissingParameterError {
-        this(apiKey, connectTimeoutMilliseconds, Constant.DEFAULT_READ_TIMEOUT_MILLISECONDS);
+        this(apiKey, connectTimeoutMilliseconds, Constant.DEFAULT_READ_TIMEOUT_MILLISECONDS, Constant.API_BASE);
+    }
+
+    /**
+     * EasyPostClient constructor.
+     *
+     * @param apiKey                     API key for API calls.
+     * @param connectTimeoutMilliseconds Timeout for connection.
+     * @param apiBase                    API base for API calls.
+     * @throws MissingParameterError
+     */
+    public EasyPostClient(String apiKey, int connectTimeoutMilliseconds, String apiBase) throws MissingParameterError {
+        this(apiKey, connectTimeoutMilliseconds, Constant.DEFAULT_READ_TIMEOUT_MILLISECONDS, apiBase);
     }
 
     /**
      * EasyPostClient constructor.
      *
      * @param apiKey                     API key for API calls.
-     * @param connectTimeoutMilliseconds timeout for connection.
-     * @param readTimeoutMilliseconds    timeout for read.
+     * @param connectTimeoutMilliseconds Timeout for connection.
+     * @param readTimeoutMilliseconds    Timeout for read.
      * @throws MissingParameterError
      */
     public EasyPostClient(String apiKey, int connectTimeoutMilliseconds, int readTimeoutMilliseconds)
             throws MissingParameterError {
+        this(apiKey, connectTimeoutMilliseconds, readTimeoutMilliseconds, Constant.API_BASE);
+    }
+
+    /**
+     * EasyPostClient constructor.
+     *
+     * @param apiKey                     API key for API calls.
+     * @param connectTimeoutMilliseconds Timeout for connection.
+     * @param readTimeoutMilliseconds    Timeout for read.
+     * @param apiBase                    API base for API calls.
+     * @throws MissingParameterError
+     */
+    public EasyPostClient(String apiKey, int connectTimeoutMilliseconds, int readTimeoutMilliseconds, String apiBase)
```

Comment:
Nit: I hate all these overloads. Have an idea for a better way, but won't block on it.

Reply:
So am I? It's Java things